onnxruntime/core/providers/stvm/stvm_utils.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef STVM_UTILS_H
+#define STVM_UTILS_H
+
+#include "stvm_common.h"
+
+#include "core/session/onnxruntime_cxx_api.h"
+#include "core/framework/ortdevice.h"
+#include "core/common/common.h"
+
+namespace onnxruntime {
+
+inline DLDataType GetDataType(ONNXTensorElementDataType type) {
+  if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_DOUBLE) {
+    return {kDLFloat, 64, 1};
+  } else if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT16) {
+    return {kDLFloat, 16, 1};
+  } else if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT) {
+    return {kDLFloat, 32, 1};
+  } else if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64) {
+    return {kDLInt, 64, 1};
+  } else if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32) {
```

Comment:
Just wondering: what about lower-precision int types, eg., for quantized arithmetic? Are they not supported?

Reply:
We plan to support full functionality of TVM including low-precision types for quantized models. But we do it step by step with testing of new features. Of course it is supported on TVM side and will be added to STVM EP on ORT side.